daily-check.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Please add an alias. @toutdesuite

Reply:
updated in https://github.com/pingcap/docs-cn/pull/3581